src/hooks/useInitialStartup.js

```diff
@@ -28,6 +28,7 @@ import {traceMetricScreen} from '../libraries/performance/firebasePerformance';
 import {useClientGetstream} from '../utils/getstream/ClientGetStram';
 import useCoreFeed from '../screens/FeedScreen/hooks/useCoreFeed';
 import StorageUtils from '../utils/storage';
+import {COLORS} from '../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `COLORS` from the theme file is a good change. It replaces hard-coded color values with dynamic ones, improving modularity and consistency.
```diff
- 
+ import {COLORS} from '../utils/theme';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->